### ISSUE TITLE
--graph-output ignores -e and -p options

### ISSUE THREAD

**Issue by socketbox** [naiquevin/pipdeptree#131]
When I use either the `-e` or `-p` flags, behavior is as expected.

When I use these flags in conjunction with `--graph-output`, it's as if `-a` were thrown.

```
bash$ pipdeptree -e ipython | grep "ipython"
bash$ pipdeptree -a | grep "ipython"
  - ipython [required: >=5.1.0, installed: 7.18.1]
      - ipython-genutils [required: Any, installed: 0.2.0]
ipython==7.18.1
    - ipython-genutils [required: Any, installed: 0.2.0]
ipython-genutils==0.2.0
  - ipython-genutils [required: Any, installed: 0.2.0]

```
![flameshot-2020-09-01-03-39-32](https://user-images.githubusercontent.com/1593835/91821731-cce7f780-ec04-11ea-964a-eaf26999ae70.png)

Another example:
```
bash$ pipdeptree -p kolibri
kolibri==0.14.4.dev0+git.20200830222656
bash$ pipdeptree -p kolibri --graph-output json | wc -l
18681
```

**Comment by naiquevin** [comment]
This has been fixed in the new version `2.0.0b1` which is a beta version, so it wouldn't get installed using `pip install pipdeptree`. 

Try `sudo pip install --pre pipdeptree` instead.

**Comment by socketbox** [comment]
Hi Vineet,

Thanks! I installed the beta and it seems as if output is consistent across formats/modes (json, stdout, raster).
